app/cmd/root.go

```diff
@@ -35,11 +38,12 @@ var rootCmd = &cobra.Command{
 	Short: "Use eBPF to speed up your Service Mesh like crossing an Einstein-Rosen Bridge.",
 	Long:  `Use eBPF to speed up your Service Mesh like crossing an Einstein-Rosen Bridge.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		s := cniserver.NewServer("/var/run/merbridge-cni.sock", "/sys/fs/bpf")
+		s := cniserver.NewServer("/host/var/run/merbridge-cni.sock", "/sys/fs/bpf")
```

Comment:
It's a good idea to make the `/host` directory configurable so that we can also debug it or use it for e2e again.

Reply:
I think this is in the helm part, but I'm not familiar with helm yet, we can do this in a follow-up